src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -181,7 +182,6 @@ class HuiGenericEntityRow extends LitElement {
       }
       .secondary,
       ha-relative-time {
-        display: block;
```

Comment:
Removed to allow ellipsis to take effect

Reply:
And it was not needed?

Reply:
I think you only wanted to remove this for `.secondary`, not for `ha-relative-time`?

Reply:
No, needs to be removed from both, since otherwise there will be no ellipsis for the ha-relative-time (ellipsis do not work with `display: block`).